docs/src/main/sphinx/connector/join-pushdown-enabled-true.fragment

```diff
@@ -24,6 +24,11 @@ join pushdown:
     session property](session-properties-definition) is
     `join_pushdown_enabled`.
   - `true`
+* - `join-pushdown.with-expressions`
+  - Enable [join pushdown](join-pushdown) with complex expressions.
```

Comment:
Do we need to have a bit more about what a "complex expression" is somewhere?

Reply:
@findepi, @Praveen2112   can you please help out here with wording?
It is important for the users to knwo the distinction between `join-pushdown.enabled` and `join-pushdown.with-expressions`